### PR TITLE
Feature/eulerian storm track

### PR DESCRIPTION
**Pull request by jeyavinoth**
I am the research tech working for James Booth (@jfbooth). I am resubmitting the Eulerian Storm Track POD (which was previously approved on the older version). This POD was submitted on the previous version of the MDTF code, so I had to make a few changes to make it compatible with the latest version.

**Checklist:**
- [x] I have reviewed my own code to ensure that if follows the [POD development guidelines](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/dev_guidelines.html)
- [x] My branch is up-to-date with the NOAA-GFDL develop branch, and all merge conflicts are resolved
- [x] The script are written in Python 3.6 or above (preferred; required if funded by a CPO grant), NCL, or R
- [x] All of my scripts are in the diagnostics/[POD short name] subdirectory, and include a main_driver script, template html, and settings.jsonc file
- [x] I have made corresponding changes to the documentation in the POD's doc/ subdirectory
- [x] If applicable, I've added a .yml file to src/conda, and my environment builds with `conda_env_setup.sh` 
- [ ] I have added any necessary data to input_data/obs_data/[pod short name] and/or input_data/model/[pod short name]
- [x] My code is portable; it uses [MDTF environment variables](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_envvars.html), and does not contain hard-coded file or directory paths
- [x] I have provided the code to generate digested data files from raw data files
- [x] Each digested data file generated by the script contains numerical data (no figures), and is 3 GB or less in size
- [x] The repository contains no extra test scripts or data files

### CODE REVIEW

File: diagnostics/eulerian_storm_track/eulerian_storm_track.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  ##########################################################
+
+  # # getting the all year standard deviation average
+  # season = 'all'
+  # time_ind = est.get_time_ind(int(os.environ['FIRSTYR']), time, season=season)
+  # std_dev = est.model_std_dev(diff, time_ind)
+  # out_file = os.environ['variab_dir']+'/eulerian_storm_track/model/PS/%s.%s.ps'%(os.environ['CASENAME'], season)
+  # plotter.plot(lonGrid, latGrid, std_dev, out_file=out_file, levels=np.arange(0,6), extend='max')
+  # out_file = os.environ['variab_dir']+'/eulerian_storm_track/model/%s.%s.png'%(os.environ['CASENAME'], season)
+  # plotter.plot(lonGrid, latGrid, std_dev, out_file=out_file, levels=np.arange(0,6), extend='max')
```
Comment (wrongkindofdoctor) [inline diagnostics/eulerian_storm_track/eulerian_storm_track.py:95]: Remove unused commented-out code

File: src/conda/env_eulerian_storm_track.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.7
+  - matplotlib=3.2.2
+  - cartopy
+  - basemap
+  - numpy 
+  - netCDF4
+  - nco
```
Comment (wrongkindofdoctor) [inline src/conda/env_eulerian_storm_track.yml:13]: What is nco required for?

File: src/conda/env_eulerian_storm_track.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+dependencies:
+  - python=3.7
+  - matplotlib=3.2.2
+  - cartopy
+  - basemap
+  - numpy 
+  - netCDF4
+  - nco 
+  - xarray
+  - mock
```
Comment (wrongkindofdoctor) [inline src/conda/env_eulerian_storm_track.yml:15]: Based on your notes, mock should no longer be required for the POD. If it is, we can add it to the base yaml

File: src/conda/env_eulerian_storm_track.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: _MDTF_eulerian_storm_track
```
Comment (wrongkindofdoctor) [inline src/conda/env_eulerian_storm_track.yml:1]: You can remove this file from the PR. Most package reqs are already in the MDTF and python3 base yamls, and we can add a couple more packages to them if needed.

File: src/conda/env_etc_composites.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+name: _MDTF_etc_composites
```
Comment (wrongkindofdoctor) [inline src/conda/env_etc_composites.yml:1]: Remove this file from the PR

File: diagnostics/eulerian_storm_track/eulerian_storm_track.py (hunk 1 above)
Comment (jeyavinoth) [reply diagnostics/eulerian_storm_track/eulerian_storm_track.py]: Done, I removed commented-out code from this and some other places. Also fixed the unused local variable and imports.

**Review by wrongkindofdoctor — CHANGES_REQUESTED** [review]

File: src/mdtf_settings.json [hunk 6]
```diff
@@ -0,0 +1,142 @@
+{
```
Comment (wrongkindofdoctor) [inline src/mdtf_settings.json:1]: Remove this file from the PR

**Review by wrongkindofdoctor — CHANGES_REQUESTED** [review]

File: src/mdtf_settings.json (hunk 6 above)
Comment (jeyavinoth) [reply src/mdtf_settings.json]: Done, removed this file.

**Review by wrongkindofdoctor — APPROVED** [review]

**Pull request merged**